### PR TITLE
added klotho analysis

### PR DESCRIPTION
**Pull request by AgusPk**
# What's this PR do?

- Added klotho analysis
- Added Readme with general information
- Added POC in go language
Solves #57 

@ulises-jeremias

<!--

Please title your PR as follows: `feature: fix foo bar`. 
Always start with the thing you are fixing, then describe the fix.
Don't use past tense (e.g. "fixed foo bar").

Explain what your PR does and why.

If you are adding a new function, please document it and add tests:

```
code you added/updated
```

If you are fixing a bug, please add a test that covers it.

Before submitting a PR, please:
  - specify the command to execute or steps to follow to know that the problem was solved

We try to process PRs as soon as possible. They should be handled within 24 hours.

Applying labels to PRs is not needed.

Thanks a lot for your contribution!

->

### CODE REVIEW

File: examples.json [hunk 1]
```diff
@@ -477,6 +477,18 @@
         "FastAPI",
         "Python"
       ]
+    },
+    {
+      "name": "Klotho",
+      "description": "A complete analysis of the service and a Proof of Concept on how to integrate it with a GO application.",
+      "url": "https://github.com/nanlabs/devops-reference/tree/main/examples/klotho-analysis/",
+      "tags": ["Examples > DevOps > Infrastructure > Pulumi > Klotho"],
```
Comment (ulises-jeremias) [inline examples.json:485]: we use the `tags` to generate the readme sections. I think this one fits better with the rest of the titles at awesome nan
 
```suggestion
      "tags": ["Examples > DevOps > Infrastructure from Code > Klotho and more!"],
```

File: examples/klotho-analysis/ANALYSIS.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Klotho
+
+## Introduction
+
+Klotho is an open-source Architecture/Infrastructure-from-Code tool designed to enable developers to build adaptable systems that can evolve over time.
+
+IMPORTANT - This analysis was made on 2023-05-16. The service was still in beta at that time.
```
Comment (ulises-jeremias) [inline examples/klotho-analysis/ANALYSIS.md:7]:
```suggestion
> IMPORTANT - This analysis was made on 2023-05-16. The service was still in beta at that time.
```

File: examples/klotho-analysis/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+We did a research on which are the Pros and Cons of using this service. You can read the full analysis [here](./ANALYSIS.md).
+
+## Proof of Concept
+
+We created a proof of concept to test the service. [here](./klotho-go-example/README.md).
+
+## Reference Documentation
+
+- [Klotho Docks](https://klo.dev/)
```
Comment (ulises-jeremias) [inline examples/klotho-analysis/README.md:31]:
```suggestion
- [Klotho Docs](https://klo.dev/)
```

**Review by ulises-jeremias — CHANGES_REQUESTED** [review]
great work! I just requested some changes. I think we need to include a `.gitignore` for the `_compiled` directory before merging. You can do that first and then I'll review the README of the example again 😊

File: examples/klotho-analysis/klotho-go-example/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# Simple Klotho app
+
+## Install Klotho
+
+### In linux
+
+Run
+
+```bash
+curl -fsSL "http://srv.klo.dev/update/latest/linux/x86_64?stream=pro:latest" -o klotho
```
Comment (di3go-martinez) [inline examples/klotho-analysis/klotho-go-example/README.md:10]: It´d be good to put the link to the official documentation where this way of installing is suggested (notice HTTP is not secured). it´d be worth trying out the docker image (https://hub.docker.com/r/klothoplatform/klotho)

File: examples/klotho-analysis/klotho-go-example/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+And then:
+
+```bash
+sudo chmod +x klotho
```
Comment (di3go-martinez) [inline examples/klotho-analysis/klotho-go-example/README.md:16]: as long as the Klotho binary is located under any user's folder, sudo cmd is not required here

File: examples/klotho-analysis/klotho-go-example/README.md (hunk 4 above)
Comment (di3go-martinez) [inline examples/klotho-analysis/klotho-go-example/README.md:10]: It´d be good to put the link to the official documentation where this way of installing is suggested (notice HTTP is not secured). it´d be worth trying out the docker image (https://hub.docker.com/r/klothoplatform/klotho)

**Review by ulises-jeremias — APPROVED** [review]

**Pull request merged**